---
# Title, summary, and page position.
linktitle: Introduction
summary: 
weight: 1
icon: book
icon_pack: fas

# Page metadata.
title: Introduction
date: ""
type: book  # Do not modify.
---

# Read!
## Before starting your metabarcoding project, here some literature you might want to read some material:

* Berg, G., Rybakova, D., Fischer, D. et al. Microbiome definition re-visited: old concepts and new challenges. Microbiome 8, 103 (2020). https://doi.org/10.1186/s40168-020-00875-0

* Abdelfattah A, Malacrin√≤ A, Wisniewski M, Cacciola SO, Schena L (2018).Metabarcoding: a powerful tool to investigate microbial communities and shape future plant protection strategies. Biological Control. [10.1016/j.biocontrol.2017.07.009](https://www.sciencedirect.com/science/article/pii/S1049964417301500)

* Earth Microbiome project. https://press.igsb.anl.gov/earthmicrobiome/

## and watch some videos:

{{< youtube kv09Mhegzuw >}}

<br/>

{{< youtube HNOMtAlTBMk >}}


# General workflow

This is the general workflow for a matabarcoding project:
1. Extract the DNA from samples.
2. Check that you have enough DNA and that it is free of unwanted contaminants.
3. Using PCR we enrich our samples for the target gene. For bacteria we usually target a portion of the 16S rRNA, for fungi we target a portion of the ITS rRNA.
4. Check for correct amplification.
5. Purify amplicons from PCR reagents.
6. Use PCR to ligate sequencing adaptors and barcodes to the amplicons.
7. Purify amplicons from PCR reagents.
8. Quantify the concentration of amplicons for each sample, and pool all samples at equimolar ratio.
9. Perform extra QC and final dilution.
10. Submit sample for sequencing.

{{% alert note%}}
If you want to know more about each single technique, you will find some material to study at the beginning of each section.
{{% /alert %}}
